.travis.yml

```diff
@@ -4,11 +4,50 @@ env:
+
 script:
+- echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
+
+- docker pull olutiab/my-heroku-build
 - docker build --target test --tag my-test-image .
+
 - docker run my-test-image tests/test_view_model.py
 - docker run my-test-image tests/test_integration.py
 - docker run -e TRELLO_API_KEY -e TRELLO_API_TOKEN my-test-image tests/test_selenium.py
+
+- docker build --target production --tag olutiab/my-heroku-build .
+
+- docker tag olutiab/my-heroku-build registry.heroku.com/olutiab-todo-app/web
+
+- docker push registry.heroku.com/olutiab-todo-app/web
+
+deploy:
+  provider: script
+  script: bash scripts/deploy.sh production $TRAVIS_TAG
+  on:
+    tags: true
+    all_branches: true
+  api_key:
+    secure: pxL7kmi1Xs5rvpkm/zAA7V6YXDo2DeFWZ5yrAvl3gsfZn4HFok5hBnW35+D66iEF9kD95VucfG8E2J12WL+5wP/kTwqTlAUjLczzi5XbaCLu+cjbIGOS8aJ6TQbK3NesXISQ0kKjufROs29XQtHLo9Dp+tMmU+pDSfDokGXwYGqopdrOqha09aa8P2CzKTGxqzZUfaOs2/Zy4xF1pAZ+t7KdAHUePHAMtIu7OLJg7dTQgkKmRK5wEbMlpadNoBtjVriuqGIXPwGnYHJwnBNXXm80jzWCUYb4/HYZGyIhZr7GPumEwfxs2pym702EM9fJzVz7mpe2rQeUbwki9T58yh7EgEH+FsxOH1ILZrK3EJnrfxvgMIa1616yAXe1r0Nk1rzFFhMzABuBwHE/RdIQWbUuFi4TAdU91pvYrc5drMQiiU/xkoGEcrnTVtAbYL7oggE9nQ9DYSekLr2Jg8V4y6kezrSVb4pG32OPkqUAo0ksBG9Kxn4F1kz+SBT8tuEuVIHdWTZDt6noBEh7TUuzzP2ObNC03PfMU81z4TG1VKPsrHSm2ZvFgOLtfi13ITtSjAbLBCe9SKNlNy3gP6q8sRjEkoLJ6/nr1VVh4dF151l+5Z0+wbVmjSk38Y6vyYOgCBMtfuWJWjcxLnLGOFjg7lVoSW3hrRJBKe3nKG9OdFw=
+  password:
+    secure: pM77RBgG322Uhr2tJdY7ChRz/LL75UwD6/Ad+1cwJtsESClcJR4044iiGOh5xYYIaJCtQk5hM6tGYtGWbJGJahZ8InTWSAUiVEhiQ/qHvxU9XXwDwFd1f6y9RVElE4PNhweKzXQC7xKa1JCe1RyqczePd4Ecf1MX4BlgYyecD6MNPixo92p1m/wGpuphaRedBPCWmnC2zc5I9meyxj1PSxFuj5n7gjoDUq29fX+fU7AHTsZvIOu7qYCtBATcu7E16RKk0mxPGwdN0HIm+6WaAUL5hYU3Orb1SoV3A2s/CsMzcOJduL9qKvpH4nyvOALn8DYx+k0lQzkLnC1QcF0il5Q1stiek8ba5k8RX49S5ZmFOBXLwK2NoPJq+Sqabo3Ry+tQEu9jY42WLk/RbXUaJyzzcQNgKL0a5ulUwemXI9XvE9Nb/EtTPS7zMb0O8SI7vcgs8zCw/H764fYWRxPThSy41QBMeHa6gJi+JnPoplp6/n9zSCdZK7prFs13N0dHB+klAg8UHNEwQC9mWnynxy6+xuFmAHuZs54YGSyUR0U/n5Q/rQZliOUgTWspStDBwben7xdVYXT+ISffy99WxmkTfHO8UgNPFI7gI91M0wqk/lYfjY6IzrbVS/CjOuJdDD49LykCNbpFbbRJyV9eM719giQNqGmEmfwghzmxsXA=
+
+      - docker login --username=olutiab --password=[SECURE] registry.heroku.com
+      
+
+      - docker push registry.heroku.com/olutiab-todo-app/web
+      - heroku container:release web --app olutiab-todo-app
+
```

Comment:
I'm pretty confident these lines won't be used atm (given you're using the `script` provider). They also seem redundant given you have the `deploy.sh` file (so might as well delete them)